tools/test/audio/process_test.sh

```diff
@@ -0,0 +1,17 @@
+#!/bin/bash
+# SPDX-License-Identifier: BSD-3-Clause
+# Copyright(c) 2020 Intel Corporation. All rights reserved.
+
+set -e
+
+# This process_test.sh is shell script wrapper for process_test.m
+# This needs octave octave-signal octave-io packages.
+
+main ()
+{
+	echo "process_test, " Component="$1", "$2", "$3", "$4", "$5"
+	octave -q --eval "[n_fail]=process_test('$1', $2, $3, $4, $5);exit(n_fail)"
+	return $?
```

Comment:
This is the default: the exit status of a function is always the exit status of the last command it ran.

Reply:
The function actually returns multiple values and here it just discard the rest.

Reply:
The `octave` process returns a single exit code. Unix does not support returning multiple exit code.

A `return $?` is always redundant just before a closing `}` (it can be useful elsewhere)

Reply:
Sorry I misunderstand the "function", `return $?` is redundant.